src/debug_commands.c

```diff
@@ -1073,6 +1091,7 @@ DEBUG_COMMAND(DocInfo) {
     return RedisModule_ReplyWithError(ctx, "Document not found in index");
   }
 
+  const bool obfuscate = (argc >= 5) && (!strcasecmp(RedisModule_StringPtrLen(argv[4], NULL), "OBFUSCATE"));
```

Comment:
That feels like breaking existing behaviour.
Not sure how okay we are with breaking `_FT.DEBUG` behaviour.

Reply:
There is no breaking of `_FT.DEBUG` commands AFAIK.
Also, this will help avoid problematic situations in which people forget to add it when executing it on a customer shard for instance while investigating.
For your consideration.